.github/workflows/e2e.yml

```diff
@@ -67,6 +67,9 @@ jobs:
           ManagerPort=2020
           LogLevel=DEBUG
           _EOF"
+          # TODO: Remove when journald logging is fixed
```

Comment:
Please add this into the file as following

bash -c "cat >> /etc/hirte/agent.conf << _EOF
ManagerPort=2020
LogLevel=DEBUG
\# TODO: Remove when journald logging is fixed
LogTarget=stderr
_EOF"
          
  And apply for other configurations

Reply:
It seems more clear to have it as completely separate command

Reply:
Anyway it's just a workaround until we find out the issue in journald logging